features/step_definitions/networking.rb

```diff
@@ -595,23 +595,23 @@
   }
 end
 
-
 Given /^the multus is enabled on the cluster$/ do
   ensure_admin_tagged
-
+  multus_status_timeout = 120
   desired_multus_replicas = daemon_set('multus', project('openshift-multus')).replica_counters(user: admin)[:desired]
   available_multus_replicas = daemon_set('multus', project('openshift-multus')).replica_counters(user: admin)[:available]
   #storing desired_multus_replicas value in desired_multus_replicas clipboard variable as well
   cb.desired_multus_replicas = desired_multus_replicas
-  unless (desired_multus_replicas == available_multus_replicas || desired_multus_replicas > env.nodes.count) && available_multus_replicas != 0
-    daemon_set('multus', project('openshift-multus')).describe(admin, quiet:false)
-    BushSlicer::Pod.get_labeled("app=multus", user: admin, project: project("openshift-multus", switch: false)) do |pod|
-      pod.describe(admin, quiet: false)
+  wait_for(multus_status_timeout) {
+    unless (desired_multus_replicas == available_multus_replicas || desired_multus_replicas > env.nodes.count) && available_multus_replicas != 0
```

Comment:
Do you have one logs for this changes?  I think we can set one multus pod to be down by manual and then run this testing to check if this works well. 
and another issue is:
Not sure if this will print a lot of logs( pod description etc.) after this changes. if so. I think we need to one way to reduce the test logs.

Reply:
@zhaozhanqi Update script to only print error message, not print pod description log in the wait_for loop.

Log from good multus status cluster:
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/4087/console

Log from bad multus status cluster:
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/4088/console